controllers/news.controller.js

```diff
@@ -78,3 +79,13 @@ exports.postCommentById = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.removeCommentById = async (req, res, next) => {
+  const { comment_id } = req.params;
+  try {
+    const deleteComment = await deleteCommentById(comment_id);
+    res.status(204).send();
```

Comment:
Can change this to res.sendStatus(204), if you dont want the empty send, but not important. Just one to know